notebooks/iceflow/0_introduction.ipynb

```diff
@@ -0,0 +1,561 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use the widget statate to build spatio-temporal parameters\n",
+    "params = ui.build_parameters()\n",
+    "params"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can query CMR to get an idea of coverages for the area we just selected.\n",
+    "# The granules and total size is an upper bound since CMR has full granules\n",
+    "# and IceFlow will subset them to only cover the area selected.\n",
+    "granules = ui.query_cmr(params=params)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Placing a data order and downloading the granules for data sets served by IceFlow\n",
+    "\n",
+    "Now that we have our constraints we just need to post our order and wait for IceFlow to fulfill it. The map widget has a button to this end, we can also put an order directly. We are going to get data from [Thwaites Glacier](https://en.wikipedia.org/wiki/Thwaites_Glacier) in Antarctica for this example (commented out there is another example for [Jakobshavn](https://en.wikipedia.org/wiki/Jakobshavn_Glacier) glacier in Greenland ). **First we are going to take a look at the granules IceFlow will subset and harmonize for us**\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 10 years of data over Thwaites glacier, Pre IceBridge, ICESat, and IceBridge data in one place!! and harmonized too!!\n",
+    "my_params ={\n",
+    "    'datasets': ['ATM1B', 'GLAH06'],\n",
+    "    'ITRF': '2014',\n",
+    "    'epoch': '2014.1',\n",
+    "    'start': '1993-11-01',\n",
+    "    'end': '2003-11-01',\n",
+    "    'bbox': '-107.4515,-75.3695,-105.3794,-74.4563'\n",
+    "}\n",
+    "\n",
+    "# Jakobshavn\n",
+    "# my_params ={\n",
+    "#     'datasets': ['ATM1B', 'GLAH06', 'ILVIS2'],\n",
+    "#     'start': '1994-01-01',\n",
+    "#     'end': '2018-12-31',\n",
+    "#     'bbox': '-107.4515,-75.3695,-105.3794,-74.4563'\n",
+    "# }\n",
+    "\n",
+    "# returns a json dictionary, the request parameters and the order's response.\n",
+    "granules_metadata = ui.query_cmr(params=my_params)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plancing the data order\n",
+    "\n",
+    "After we place an order in IceFlow a few things will happen, first we'll get a notification email that our order has been accepted and is being processed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "orders = ui.place_data_orders(params=my_params)\n",
+    "orders"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Downloading the data\n",
```

Comment:
The output naming convention is not ideal - It's really hard to know what output file came from what data set. Maybe this is a Hermes enhancement? The output files should retain the input file name for provenance.